<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewImage</class>
 <widget class="QDialog" name="DialogNewImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New image</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,3,20,0" columnstretch="1,1" rowminimumheight="1,1,1,1">
   <item row="2" column="0">
    <widget class="QGroupBox" name="grpLevelDesignerParams">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Image parameters</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>371</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="grdParams"/>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Info</string>
     </property>
     <widget class="QLabel" name="lblInfo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>331</width>
        <height>221</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="btnResult">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>New image:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnResult</sender>
   <signal>accepted()</signal>
   <receiver>DialogNewImage</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnResult</sender>
   <signal>rejected()</signal>
   <receiver>DialogNewImage</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
